bankholidays/src/main/java/com/celements/calendar/bankholidays/classes/BankHolidayClassesRole.java

```diff
@@ -0,0 +1,16 @@
+package com.celements.calendar.bankholidays.classes;
+
+import org.xwiki.component.annotation.ComponentRole;
+
+import com.celements.model.classes.ClassDefinition;
+
+@ComponentRole
+public interface BankHolidayClassesRole extends ClassDefinition {
+
+  public static final String DATE_FORMAT_TIMESTAMP = "dd.MM.yyyy HH:mm";
+  public static final String VALIDATION_REGEXP_DATE_TIME = "/^(0[1-9]|[12][0-9]|3[01])\\."
+      + "(0[1-9]|1[012])\\.(19|20|99)\\d\\d (0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]$/";
+  public static final String VALIDATION_MESSAGE_DATE_TIME = "bank_holiday_validation_date_time";
+  public static final String VALIDATION_REGEXP_HOLIDAY_NAME = "/^[a-zA-Z]{0,64}$/";
```

Comment:
You can leave the details on the characters. I don't see a reason why we shouldn't allow names as e.g. "Christmas / Hanukkah". So just use "." instead of "[a-zA-Z]"